.dockerignore

```diff
@@ -1,3 +1,7 @@
 .build/
 .swiftpm/
 SPI-checkouts/
+Public/build
+Public/node_modules
+Public/.config.codekit3
+Public/package*.json
```

Comment:
We're currently serving these files: https://swiftpackageindex.com/package.json but they're just inputs for CodeKit. Adding them here will make sure they're skipped during

```
COPY --from=build /build/Public /run/Public
```

I _think_ this is all we need/want to skip but please double check!

Reply:
You're absolutely right, none of those are needed. I just checked and everything else in the Public folder *is* needed. I really wish we could split these up a bit, but it wasn't easy to do with CodeKit.